components/Downloads/Release/LinkWithArrow.tsx

```diff
@@ -11,7 +11,7 @@ const LinkWithArrow: FC<PropsWithChildren<AccessibleAnchorProps>> = ({
 }) => (
   <Link {...props}>
     {children}
-    <ArrowUpRightIcon className="ml-1 inline w-3 fill-white" />
+    <ArrowUpRightIcon className="ml-1 inline w-3 fill-shadow dark:fill-white" />
```

Comment:
```suggestion
    <ArrowUpRightIcon className="ml-1 inline w-3 fill-neutral-600 dark:fill-white" />
```

The icon color for the white theme in [Figma](https://www.figma.com/file/pu1vZPqNIM7BePd6W8APA5/Node.js?type=design&node-id=332-34678&mode=design&t=PJg8dWIFUeV6eI3N-0) is `neutral-600`, can you update it with these 🙇